scripts/build/static-server.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const colors = require('colors');
+const express = require('express');
+const fs = require('fs');
+const serveIndex = require('serve-index');
+
+/**
+ * An Express-based Web server for static files.
+ * Automatically generates a readable directory listing when no index.html file is present.
+ */
+class StaticServer {
+  /**
+   * @param {!PathResolver} pathResolver
+   * @param {!express=} expressLib
+   * @param {!fs=} fsLib
+   * @param {!serveIndex=} serveIndexLib
+   */
+  constructor({
+    pathResolver,
+    expressLib = express,
+    fsLib = fs,
+    serveIndexLib = serveIndex,
+  } = {}) {
+    this.pathResolver_ = pathResolver;
+    this.expressLib_ = expressLib;
+    this.fsLib_ = fsLib;
+    this.serveIndexLib_ = serveIndexLib;
+  }
+
+  /**
+   * Starts a static file server asynchronously and returns immediately.
+   * @param {string} path Relative or absolute path to a filesystem directory to serve
+   * @param {number} port
+   * @param {!Array<string>=} fileExtensions Whitelisted file extensions to show in the directory index
+   * @param {string} stylesheetAbsolutePath Absolute path to a CSS file containing styles for the directory index
```

Comment:
Yes, they all have the same CSS file.

It's not about making it customizable, it's about keeping all (or most) of the file paths in one place (`webpack.config.js`) so that our file structure is easier to understand and maintain.